corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -417,7 +418,9 @@ def synclog_to_sql_object(synclog_json_object):
     return synclog
 
 
+@architect.install('partition', type='range', subtype='date', constraint='day', column='date')
```

Comment:
still needs a migration to make DB changes

Reply:
I missed committing that part. Will update the main PR